_1327/documents/templates/documents_edit.html

```diff
@@ -403,6 +403,26 @@ <h4 class="modal-title" id="deleteAutosaveHeader">{% trans "Do you really want t
 		// add datepicker to date inputs
 		$("input[name*='date']:not([readonly='True'])").datepicker( $.datepicker.regional["{{ LANGUAGE_CODE }}"] );
 
+		function getCurrentTime() {
+			const currentDate = new Date();
+			const currentHour = currentDate.getHours().toString();
+			let currentMinute = currentDate.getMinutes().toString();
+			if(currentMinute.length == 1) currentMinute = "0" + currentMinute;
```

Comment:
This is certainly nitpicking, but I prefer putting a space between control structures and their parenthesis to distinguish them from function calls. Quickly looking over other parts of this file, it seems like this is the preferred formatting in this project.